src/lib/button-toggle/button-toggle.md

```diff
@@ -18,5 +18,12 @@ and `ngModel` is not supported.
 The button-toggles will present themselves as either checkboxes or radio-buttons based on the 
 presence of the `multiple` attribute. 
 
+For button toggles with icons only, each button toggle should be marked with a meaningful label by 
```

Comment:
I would rephrase `"with icons only"` to `"containing only icons"` here and elsewhere